api/address/routes.go

```diff
@@ -39,40 +40,88 @@ func Routes(router *wrapper.RouterWrapper) {
 func GetAccount(c *gin.Context) {
 	ef, ok := c.MustGet("elrondFacade").(FacadeHandler)
 	if !ok {
-		c.JSON(http.StatusInternalServerError, gin.H{"error": errors.ErrInvalidAppContext.Error()})
+		c.JSON(
+			http.StatusInternalServerError,
+			core.GenericAPIResponse{
```

Comment:
Why not move inside api package? Do we need this elsewhere? Core will get full of unrelated stuff.

Reply:
first time I placed them in the API package but there were cyclic imports

Reply:
created a sub-package `shared` inside the `api` package